### PR TITLE
Add testing examples

### PR DESCRIPTION
**Pull request by vbrazo**
This PR is related to https://github.com/v-community/v_by_example/issues/42.

### CODE REVIEW

**Review by dhonx — APPROVED** [review]
So far LGTM... I think just merge it... then we will review later... I think ??

Great Work !!!  👏 👌

File: examples/testing.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Testing in software development is a process that aims to evaluate the functionality of an application with an intent to find whether the code met the specified requirements or not as well as identifying the problems to ensure that the product has expected quality.
+
+## Automated tests
+
+Automated tests follow the process of testing the software using an automation tool to find the defects. In this process, programmers execute the test scripts and generate the test results automatically by using automation tools.
+
+## Tests in V
+
+In V, all test files have to be named with following format: `*_test.v` and the functions should start with `test_`.
```
Comment (dhonx) [inline examples/testing.md]: I think just add the word `the` before `following`
> In V, all test files have to be named with the following format: `*_test.v` and the functions should start with `test_`.

**Review by dhonx — CHANGES_REQUESTED** [review]

File: examples/testing.md (hunk 1 above)
Comment (vbrazo) [reply examples/testing.md]: sounds good! Adding now.

**Review by dhonx — APPROVED** [review]
Great work !!!  👏 👌 💯

**Pull request merged**